src/styles/rhd-theme/components/_cards.scss

```diff
@@ -309,7 +352,7 @@
 }
 
 .rhd-c-card__footer:not(:last-child) {
-  padding-bottom: var(--pf-global--spacer--sm); // 8px
+  padding-bottom: var(--pf-global--spacer--md); // 8px
```

Comment:
Commented size is incorrect - should be `// 16px`

Reply:
done